### PR TITLE
Update/channel page UI

### PR DESCRIPTION
**Pull request by codesagecoder**

### CODE REVIEW

File: components/ChannelDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  }, [channelId])
+
+  const moderators = useMemo(() => {
+    const users = [];
+
+    !!channel?.hosts?.[0] && users.push(channel.hosts[0]);
+    !!channel?.moderator && users.push(channel.moderator);
+
+    return users;
+  }, [channel]);
```
Comment (techeng322) [inline components/ChannelDetails/index.tsx:28]: can we move this into hook as named `useChannelDetails`?

File: components/Sidebar.tsx [hunk 2]
```diff
@@ -6,16 +6,19 @@ import { Separator } from './ui/separator';
...
 
 export default function MobileMenu({ isSingleCast = false }: { isSingleCast?: boolean }) {
   const { user, signOut } = useNeynarProvider();
 
   return (
     <div className="flex h-full flex-col gap-4 md:px-16 md:py-6">
-      <a href="/" className="mb-8 flex items-center gap-2 max-md:hidden">
+      <Link href="/" className="mb-8 flex items-center gap-2 max-md:hidden">
         <Image src="/images/notes.png" width={20} height={20} alt="" />
         <span className="font-semibold">Sonata</span>
```
Comment (techeng322) [inline components/Sidebar.tsx:19]: thanks for using `<Link />`🙇

**Review by techeng322 — APPROVED** [review]

File: components/Header/HomeButton.tsx [hunk 3]
```diff
@@ -1,29 +1,13 @@
...
-          strokeLinejoin="round"
-        />
-      </svg>
+    <Link href='/' className={cn("flex items-center gap-1", className)}>
+      <ChevronLeft/>
       <p className="font-sora text-[18px] font-semibold">Home</p>
-    </button>
+    </Link>
   );
 };
```
Comment (SweetmanTech) [inline components/Header/HomeButton.tsx:12]: return is redundant right?
- remove `return` 🚮 
- remove curly braces `{}` 🚮 

same functionality with less code right?

File: components/Sidebar.tsx (hunk 2 above)
Comment (SweetmanTech) [reply components/Sidebar.tsx:19]: +++

**Review by SweetmanTech — APPROVED** [review]

**Pull request merged**